client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-fallback-credentials/components/credentials-form.tsx

```diff
@@ -0,0 +1,85 @@
+import { NextButton } from '@automattic/onboarding';
+import { useTranslate } from 'i18n-calypso';
+import { FC } from 'react';
+import Notice from 'calypso/components/notice';
+import { useQuery } from 'calypso/landing/stepper/hooks/use-query';
+import { ErrorMessage } from '../../site-migration-credentials/components/error-message';
+import { PasswordField } from '../../site-migration-credentials/components/password-field';
+import { SpecialInstructions } from '../../site-migration-credentials/components/special-instructions';
+import { UsernameField } from '../../site-migration-credentials/components/username-field';
+import { useCredentialsForm } from '../hooks/use-credentials-form';
+
+interface CredentialsFormProps {
+	onSubmit: ( from?: string ) => void;
+	onSkip: () => void;
+}
+
+export const CredentialsForm: FC< CredentialsFormProps > = ( { onSubmit, onSkip } ) => {
+	const translate = useTranslate();
+	const { control, errors, isBusy, submitHandler, canBypassVerification } =
+		useCredentialsForm( onSubmit );
+
+	const queryError = useQuery().get( 'error' ) || null;
+
+	let errorMessage;
+	if ( errors.root && errors.root.type !== 'manual' && errors.root.message ) {
+		errorMessage = errors.root.message;
+	} else if ( queryError === 'ticket-creation' ) {
+		errorMessage = translate(
+			'We ran into a problem submitting your details. Please try again shortly.'
+		);
+	}
+
+	const getContinueButtonText = () => {
+		if ( isBusy && ! canBypassVerification ) {
+			return translate( 'Verifying credentials' );
+		}
+		if ( canBypassVerification ) {
+			return translate( 'Continue anyways' );
```

Comment:
I believe this should read "Continue anyway"

Reply:
Sure, just updated it!